cmd/gcp-controller-manager/app/node_annotater.go

```diff
@@ -245,11 +248,11 @@ func parseNodeURL(nodeURL string) (project, zone, instance string, err error) {
 }
 
 // TODO: move this to instance.Labels. This is gross.
-func extractKubeLabels(instance *compute.Instance) (map[string]string, error) {
+func extractKubeLabels(instance *compute.Instance) (map[string]string, bool, error) {
```

Comment:
Define `errNoMetadata` instead of adding a bool. Use `return nil, errNoMetadata` instead of `return nil, false, nil`